src/styles/partials/components/_step-list.scss

```diff
@@ -106,6 +106,9 @@
 .dg_step-list.static {
   .dg_step-list__toggle-btn {
     cursor: default;
```

Comment:
Question for the group. Since the user is still able to open and close each item in the static step list, should the `cursor` property be set to `pointer` to communicate that functionality to the user?

Reply:
actually @theilman7 the static step list does not allow the user to interact. All steps are expanded and no buttons are visible.